UndertaleModTool/Scripts/Resource Repackers/ImportASound.csx

```diff
@@ -211,7 +211,7 @@ if (dialogRet == DialogResult.OK)
             File        = Data.Strings.MakeString(fname),
             Effects     = 0,
             Volume      = 1.0F,
-            Pitch       = 1.0F,
+            Pitch       = 0,
```

Comment:
I think this is a redundant change that you should undo.
```suggestion
            Pitch       = 1.0F,
```

Reply:
You should undo this, and instead create a separate PR with only this particular change.